src/js/components/Grommet/doc.js

```diff
@@ -21,6 +21,14 @@ export const doc = Grommet => {
     full: PropTypes.bool
       .description('Whether to take the whole viewport.')
       .defaultValue(false),
+    options: PropTypes.object
+      .description(
+        `Opt in that would assure that the id is placed on 
```

Comment:
It seems like this is being set up to potentially allow other options/specifications in the future. What about something like (probably needs some refinement):

Any other options to apply globally throughout your application. If singleId is true, any `id` placed on Layer will only apply to the DOM node associated with the visual portion of the Layer and any `id` placed on Select will only apply to its underlying input. This opt-in flag is intended to resolve accessibility considerations that `id` should be unique and placed on a single DOM node while maintaining backwards compatibility with the existing behavior.

Reply:
^^ or if this PR isn't going to address Select, then maybe remove the Select stuff I mention above.